roles/reproducer/tasks/prepare_networking.yml

```diff
@@ -143,6 +143,44 @@
       server=/{{ _domain }}/127.0.0.2
       server=/apps-crc.testing/127.0.0.2
       server=/crc.testing/127.0.0.2
+      server=/utility/127.0.0.2
+
+- name: Feed utility records
+  block:
+    # TODO: feed IPv6 data in the record
+    - name: Build utility records dataset
+      when:
+        - hostvars[host].ansible_default_ipv4.address | default('') | length > 0
+      vars:
+        _host_record:
+          - names:
+              - "{{ host }}.utility"
+            ips:
+              - "{{ hostvars[host].ansible_default_ipv4.address | default('') }}"
```

Comment:
Don't we have a map of what IPs is on each network for hosts? If it is not including the hypervisor bridge ip - we could add it? And then instead of hostvars default route interface. We go `our_map[host][pub_net][ip_address]` ?

I think it would make our life simpler if we used one of the bridges, for example the "pub_net" bridge ip address for the "hypervisor". Stay inside the lab - the public interface on the hypervisor is not where we want traffic to terminate?

Reply:
Also, do we need to put entries for controller-0, compute, ocp, etc on the utlity domain? It will point to same address that we already have in `compute-0.ocp.openstack.lab` no? Will it get confusing to get search domains good in resolv.conf?

Reply:
Lemme check what we have in the networking_mapper. Using the "gateway" might be something, though we'll have to ensure it's properly mapped on an NFV deployment.

As for "controller, compute, ocp" in the utility "zone": no issue afaik, because `.utility` isn't added to the `domain-search` - meaning it's mostly "unknown". At least that's my understanding, but I'm slightly rusted with DNS, didn't play too much for the past 5-6 years... Lemme deploy, do some testing, and we can also sync live on that matter.

Reply:
I think I solved this by consuming the content of the networking_mapper instead. Lemme know if this matches your ideas?
I used "ctlplane" since this network is always present and "local" to the lab. "public" wouldn't really match the need here, nor "ocppr" - or, at least, they aren't listed in the mapper, meaning not that easy to get.